web/packages/shared/components/AuthorizeDeviceWeb/AuthorizeDeviceWeb.tsx

```diff
@@ -0,0 +1,120 @@
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Box, Flex, ButtonPrimary, Text, ButtonLink } from 'design';
+import { Link } from 'react-router-dom';
+
+import cfg from 'teleport/config';
+
+import {
+  DownloadConnect,
+  DownloadLink,
+} from 'shared/components/DownloadConnect/DownloadConnect';
+
+export const DeviceTrustConnectPassthrough = ({
+  authorizeWebDeviceDeepLink,
+  downloadLinks,
+}: {
+  authorizeWebDeviceDeepLink: string;
+  downloadLinks: Array<DownloadLink>;
+}) => {
+  return (
+    <Box
+      css={`
+        text-align: center;
+        line-height: 32px;
+      `}
+    >
+      <Flex flexDirection="column">
+        <Text fontWeight={300} fontSize={7} mb={7}>
+          Click{' '}
+          <span
+            css={`
+              font-weight: 700;
+            `}
+          >
+            Open Teleport Connect
+          </span>{' '}
+          in the dialog in your browser
+        </Text>
+        <Text fontSize={7} mb={10} fontWeight={300}>
+          If you don't see a dialog, press{' '}
+          <span
+            css={`
+              font-weight: 700;
+            `}
+          >
+            Launch Teleprt Connect
+          </span>{' '}
+          below
+        </Text>
+        <Flex justifyContent="center" mb={9}>
+          <ButtonPrimary
+            textTransform="none"
+            width="280px"
+            as="a"
+            href={authorizeWebDeviceDeepLink}
+          >
+            Launch Teleport Connect
+          </ButtonPrimary>
+        </Flex>
+        <Box>
+          <Text fontSize={3}>
+            Don't have Teleport Connect yet?{' '}
+            {downloadLinks.length < 2 ? (
+              <ButtonLink
+                css={`
+                  text-decoration: none;
+                  font-size: 16px;
+                  color: ${props => props.theme.colors.brand};
+                `}
+                as="a"
+                href={downloadLinks[0].url}
+              >
+                Download it now
+              </ButtonLink>
+            ) : (
+              <DownloadConnect downloadLinks={downloadLinks} />
+            )}
+          </Text>
+        </Box>
+        <Box
+          css={`
+            text-align: center;
+            width: 100%;
+            position: absolute;
+            bottom: 24px;
+          `}
+        >
```

Comment:
It does work, and I suppose it can be better. I've never been a fan of having what looks like a "component" but really its just wrapped css somewhere else in the file. I prefer seeing the css explicitly in the thing I'm designing. However, I can update if we wanna keep a standard

Reply:
I'm the opposite, I prefer styled components wrap other styled components instead of inline CSS in the JSX (otherwise we might as well use Tailwind)

Reply:
> we might as well use Tailwind

Don't tease me with a good time.

I'll update to wrapped. I think devving with the inline works and once I'm happy with changes, moving to a wrapped is a good workflow for me 👍

Reply:
I wrapped them all except link since it was a one-liner